### PR TITLE
moved css files up to main directory

### PR DESCRIPTION
**Pull request by kdclang**

### CODE REVIEW

**Comment by kdclang** [comment]
That's basically what I did. I moved all the css files and references within the tutorials to the main directory level, so there is one set of css files, rather than a set for each subfolder.

File: css/style.css [hunk 1]
```diff
@@ -9,9 +9,11 @@ table {
 .ui-draggable, .ui-droppable {
 	background-position: top;
 }
-
+a {
+	color:blue !important;
+}
 p.small {
-   line-height: 
+	line-height:
```
Comment (ewpatton) [inline css/style.css]: We should either fix or remove this incomplete property specification. (I know that wasn't part of the original intent of this PR).

File: css/style.css [hunk 2]
```diff
@@ -30,146 +32,110 @@ a.ui-button:active,
...
-  color: #8fc202;
-  font-size: 12px;
+.control {
+	background-color: #b18e35;
+	border: 1px dashed #000;
+	color: white;
+	padding:1px;
 }
 
-.panel-text {
```
Comment (ewpatton) [inline css/style.css]: The panel-text class is also referenced in blocks/blocks.html and howtos/howtos.html. I'm not sure that we want to remove these two rules from this CSS file.

File: css/style.css [hunk 3]
```diff
@@ -30,146 +32,110 @@ a.ui-button:active,
...
-  background-color: #f3f3f3;
-  padding: 10px;
-  margin-right: 10px;
-  margin-left: 10px;
+.right {
+	float:right;
+	max-width: 30%;
 }
 
 /* Enlarge Image */
```
Comment (ewpatton) [inline css/style.css]: In yr/css/style.css, the `.tutorialDiv` class also had `display: inline-block`. This version does not. I think this is the right way to do it (divs are block by default, not inline-block), which gives the appropriate vertical arrangement of sections.

File: css/style.css [hunk 4]
```diff
@@ -204,155 +171,46 @@ body {
   opacity: 1;
```
Comment (ewpatton) [inline css/style.css]: From malden/css/style.css, the follow rule is missing:

```css
img.DialogBox-image {
  max-width: 100%;
}
```

File: yr/tutorials/ball_bounce.html [hunk 5]
```diff
@@ -31,12 +31,13 @@
           //$( "button, input, a" ).click( );
         } );
   </script>
+  <!--
```
[the remark this answers is not in the document]
Comment (kdclang) [reply yr/tutorials/ball_bounce.html]: done

File: yr/tutorials/ball_bounce.html [hunk 6]
```diff
@@ -49,7 +50,7 @@
   </script>
 </head>
 <body>
-
+<!--
```
[the remark this answers is not in the document]
Comment (kdclang) [reply yr/tutorials/ball_bounce.html]: done

File: yr/tutorials/ball_bounce.html [hunk 7]
```diff
@@ -175,8 +177,9 @@ <h4>Extend your app</h4>
       <div class="tutorialNextButtonContainer" style="width:100%;text-align:center;">
         <button type="button" class="btn btn-success btn-lg tutorialNextButton" style="font-size:24pt;">Next Step</button>
       </div>
-      </div>
+    </div>
 
+      <div id="howToAccordion" class="accordion" style="display:none"></div>
```
[the remark this answers is not in the document]
Comment (kdclang) [reply yr/tutorials/ball_bounce.html]: Yes, a mistake. I thought I needed it with the howto stuff. Removed now.

File: yr/tutorials/cat_map.html [hunk 8]
```diff
@@ -26,7 +26,7 @@
 <body>
 
 <div style="width:100%;height: 50px;text-align:center;background-color: #f6f6f6;margin-bottom: 10px;">
-<div id="youthMobilePowerTab" style="float:left;height:40px;width:50%;background-color: #ccdea8;padding-top: 5px;padding-bottom: 5px;">Youth Mobile Power</div>
+<div id="youthMobilePowerTab" style="float:left;height:40px;width:50%;background-color: #ccdea8;padding-top: 5px;padding-bottom: 5px;">Tutorial</div>
```
[the remark this answers is not in the document]
Comment (kdclang) [reply yr/tutorials/cat_map.html]: Confirming all changes. Turns out several issues in the tutorial (which is not used by YR anyway) so I fixed it up as I went through.

File: yr/tutorials/digital_doodle.html [hunk 9]
```diff
@@ -31,12 +31,13 @@
           //$( "button, input, a" ).click( );
         } );
   </script>
+  <!--
```
[the remark this answers is not in the document]
Comment (kdclang) [reply yr/tutorials/digital_doodle.html]: done

File: yr/tutorials/digital_doodle.html [hunk 10]
```diff
@@ -49,7 +50,7 @@
   </script>
 </head>
 <body>
-
+<!--
```
[the remark this answers is not in the document]
Comment (kdclang) [reply yr/tutorials/digital_doodle.html]: done

File: yr/tutorials/digital_doodle.html [hunk 11]
```diff
@@ -175,7 +177,8 @@ <h4>Extend your App!</h4>
   </div>
 
       </div>
-
+    
+    <div id="howToAccordion" class="accordion" style="display:none">
```
[the remark this answers is not in the document]
Comment (kdclang) [reply yr/tutorials/digital_doodle.html]: removed now

File: yr/tutorials/healthy_plant.html [hunk 12]
```diff
@@ -23,25 +23,44 @@
...
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-31803258-2', 'auto');
+  ga('send', 'pageview');
+
+</script>
 </head>
 <body>
-
+<!--
```
[the remark this answers is not in the document]
Comment (kdclang) [reply yr/tutorials/healthy_plant.html]: done

File: yr/tutorials/mapTheMovement.html [hunk 13]
```diff
@@ -343,8 +343,10 @@ <h3>Use a TextBox</h3>
 
   <h3>Map Properties</h3>
   <div class="setup" id="mapProperties"></div>
+   
 
 </div>
+</div>
```
[the remark this answers is not in the document]
Comment (kdclang) [reply yr/tutorials/mapTheMovement.html]: removed

File: yr/tutorials/mytodolist.html [hunk 14]
```diff
@@ -24,12 +24,13 @@
       $( ".widget input[type=submit], .widget a, .widget button" ).button();
     } );
   </script>
-
+<!--
```
[the remark this answers is not in the document]
Comment (kdclang) [reply yr/tutorials/mytodolist.html]: done

File: yr/tutorials/talk_to_me.html [hunk 15]
```diff
@@ -31,12 +31,13 @@
           //$( "button, input, a" ).click( );
         } );
   </script>
+  <!--
```
[the remark this answers is not in the document]
Comment (kdclang) [reply yr/tutorials/talk_to_me.html]: done

File: css/style.css (hunk 1 above)
Comment (kdclang) [reply css/style.css]: I added a .9em to complete the property.

File: css/style.css (hunk 2 above)
Comment (kdclang) [reply css/style.css]: For those tutorials in main directory, I believe all referenced via index.html, I renamed css/style.css to css/mainstyle.css and referenced that from index.html. I believe blocks.html and howto.html are loaded through index, so should use mainstyle.css and get those classes that way.

File: css/style.css (hunk 3 above)
Comment (kdclang) [reply css/style.css]: added display: inline-block;

File: css/style.css (hunk 4 above)
Comment (kdclang) [reply css/style.css]: added at end of style.css

File: css/style.css (hunk 2 above)
Comment (ewpatton) [reply css/style.css]: I doubled checked and believe that you are correct.

**Pull request merged**